util/response_request.py

```diff
@@ -0,0 +1,30 @@
+from util.str_utils import preprocess_text
+import requests
+
+
+def request_hero_name_set():
+    WEB_REQUEST = requests.get('https://dota2.gamepedia.com/api.php?' +
+                                'action=cargoquery&tables=heroes&fields=title&where=game'+
+                                '+IS+NULL&limit=500&format=json')
+    WEB_JSON = WEB_REQUEST.json()
+    SELECTED_HERO_RESPONSES = {}
```

Comment:
What purpose does SELECTED_HERO_RESPONSES serve here?

Reply:
In the comments I saw that responses were hard coded to increase flexibility. This is just an additional set that can be added to in order to exclude heroes or items (heros in this case) from being included in the set. 

Say you wanted a hero name to not be in your excluded set, you would add it there. Not needed but thought it could help address any flexibility issues that could arise in the future.

Reply:
Removed in https://github.com/Jonarzz/DotaResponsesRedditBot/pull/41/commits/60c457340a9fc231d6bd8ad0387d2898d82ce3a1

When ahead and removed the feature. It was a novel idea but I think it adds unneeded complexity and if an issue ever arises it can be reincorporated.